server/Groups/Groups.js

```diff
@@ -5,6 +5,58 @@ const Groups = require('../../db/Groups/Groups.js');
 const GroupMembers = require('../../db/GroupMembers/GroupMembers.js');
 const phone = require('phone');
 
+exports.updateGroupPrivacy = (req, res) => {
```

Comment:
Can this also be written to be more generic? To be able to update anything about the group rather than specifically their privacy settings?